ocaml/xapi/workload_balancing.ml

```diff
@@ -675,7 +675,7 @@ let get_evacuation_recoms ~__context ~uuid =
       []
     else
       match inner_xml with
-      | (Xml.Element ( _, _, [Xml.Element("CanPlaceAllVMs",_,_)])) ->
+      | (Xml.Element ( _, _, [_;_;Xml.Element("CanPlaceAllVMs",_,_)])) ->
```

Comment:
Agreed: I'm sorry but I don't understand what the commit-message means.
Also its first line (subject-line) is awfully long.